src/libs/UnreadIndicatorUpdater/index.js

```diff
@@ -3,12 +3,13 @@ import Onyx from 'react-native-onyx';
 import ONYXKEYS from '../../ONYXKEYS';
 import updateUnread from './updateUnread/index';
 import * as ReportUtils from '../ReportUtils';
+import CONST from '../../CONST';
 
 Onyx.connect({
     key: ONYXKEYS.COLLECTION.REPORT,
     waitForCollectionCallback: true,
     callback: (reportsFromOnyx) => {
-        const unreadReports = _.filter(reportsFromOnyx, ReportUtils.isUnread);
+        const unreadReports = _.filter(reportsFromOnyx, (report) => ReportUtils.isUnread(report) || report.notificationPreference !== CONST.REPORT.NOTIFICATION_PREFERENCE.HIDDEN);
```

Comment:
I think this needs to be AND
```suggestion
        const unreadReports = _.filter(reportsFromOnyx, (report) => ReportUtils.isUnread(report) && report.notificationPreference !== CONST.REPORT.NOTIFICATION_PREFERENCE.HIDDEN);
```

Reply:
got even more unread chats 😂 

<img width="183" alt="image" src="https://github.com/Expensify/App/assets/36083550/323f23b3-4a3a-4a00-b4f0-bf8ddefd903b">

Reply:
haha yes I didn't test this yet. you are right.

Reply:
`Not Read To Build`

Reply:
haha I was `too quick to build`